src/test/polymer/polymer2-element-scanner_test.ts

```diff
@@ -371,6 +371,18 @@ namespaced name.`,
         mixins: ['Mixin'],
         warningUnderlines: [],
       },
+      {
```

Comment:
is the changed behavior still covered by current tests? There are a lot of edge cases handled above that should be tested as well.